### PR TITLE
make new feature teaser edition dependent

### PR DESCRIPTION
**Pull request by machisuji**
WP [#32100](https://community.openproject.com/projects/openproject/work_packages/32100/activity)

### CODE REVIEW

File: frontend/src/app/components/homescreen/blocks/new-features.component.ts [hunk 1]
```diff
@@ -30,7 +30,6 @@ import {Component} from '@angular/core';
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {DomSanitizer} from "@angular/platform-browser";
-import {dashboardWebsiteUrl} from "core-app/modules/dashboards/dashboard-constants.const";
```
Comment (openprojectci) [inline frontend/src/app/components/homescreen/blocks/new-features.component.ts:33]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/66b03456e70187b84d37c93707751724e22976cd/frontend/src/app/components/homescreen/blocks/new-features.component.ts#L33"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>

File: config/locales/js-en.yml [hunk 2]
```diff
@@ -230,13 +230,31 @@ en:
...
-                                      <li>task progress,</li>
-                                      <li>tasks assigned to users.</li></ul>"
-          learn_about: "Learn more about the new status widgets"
+          learn_about: "Learn more about the new features"
+          standard:
+            learn_about_link: https://www.openproject.org/openproject-10-4
+            current_new_feature_html: >
+              The release contains various new features and improvements: <br><br>
+              <ul>
+                <li>Default <b>design templates</b> to select a light or dark theme (premium feature)</li>
```
Comment (HDinger) [inline config/locales/js-en.yml]: Each of them is missing a dot at the end.

File: frontend/src/app/components/homescreen/blocks/new-features.component.ts [hunk 3]
```diff
@@ -39,12 +38,12 @@ import {dashboardWebsiteUrl} from "core-app/modules/dashboards/dashboard-constan
...
 
     <div class="widget-box--description">
-      <p [innerHtml]="text.currentNewFeatureHtml"></p>
+      <p [innerHtml]="currentNewFeatureHtml"></p>
 
       <a class="widget-box--teaser-image"></a>
     </div>
 
-    <a [href]="teaserWebsiteUrl()" target="_blank">{{ text.learnAbout }}</a>
+    <a [href]="teaserWebsiteUrl" target="_blank">{{ text.learnAbout }}</a>
```
Comment (HDinger) [inline frontend/src/app/components/homescreen/blocks/new-features.component.ts:46]: The text is still wrong as it is not prefixed.

File: config/locales/js-en.yml [hunk 4]
```diff
@@ -230,13 +230,31 @@ en:
...
+          bim:
+            learn_about_link: https://www.openproject.org/openproject-bim-10-4
+            current_new_feature_html: >
+              OpenProject contains a <b>new IFC module to integrate 3D models</b>
+              into your project management application and to work collaboratively
+              with these files with your team without installation of separate
+              costly software or licenses. <br><br>
+              <ul>
+                <li><b>Import IFC files</b> of building models into your OpenProject application</li>
+                <li>overview of work package status,</li>
```
Comment (HDinger) [inline config/locales/js-en.yml]: The other text needs to be changed, too.

**Review by HDinger — COMMENTED** [review]
Looks good 👍 

Please update the latest image for the BIM block, as well as the text. For the standard block there is also already an image uploaded in the ticket. And please also have a look at the CodeClimate Issues. :)

File: frontend/src/app/components/homescreen/blocks/new-features.component.ts (hunk 3 above)
Comment (machisuji) [reply frontend/src/app/components/homescreen/blocks/new-features.component.ts:46]: The learn about text is the same for both cases. Or what do you mean? @HDinger

**Comment by machisuji** [comment]
@HDinger I think I've addressed everything including the code climate issues that made sense for this PR.

File: frontend/src/app/components/homescreen/blocks/new-features.component.ts (hunk 3 above)
Comment (HDinger) [reply frontend/src/app/components/homescreen/blocks/new-features.component.ts:46]: I thought this one was also prefixed, but you are right. They use the same string. So just ignore my comment :)

**Review by HDinger — APPROVED** [review]
👍 Good to merge, once travis agrees.

**Pull request merged**